src/js/layout.js

```diff
@@ -27,7 +27,22 @@ export default class layout {
         }
 
         // wrap the output in a form-group div & return
-        const className = data.id ? `fb-${data.type} form-group field-${data.id}` : ''
+        let className = data.id ? `fb-${data.type} form-group field-${data.id}` : ''
+    
+        // Lift any col- and row- type class to the form-group wrapper. The row- class denotes the row group it should go to
+        if(data.className){
+          const classes = data.className.split(' ').filter(className => (className.startsWith('col-') || className.startsWith('row-')));
+          if(classes && classes.length > 0){
+            className += ` ${classes.join(' ')}`;
+
+            // Now that the col- types were lifted, remove from the actual input field
```

Comment:
I'm not sure if this is working correctly.
![image](https://user-images.githubusercontent.com/1457540/48185753-df37c980-e2ea-11e8-9b87-da98a0e3ccca.png)

Reply:
So this implementation actually only applies to formRender not formBuilder. I did not know how to make it render with columns on the builder page, only once rendered. There is a disconnect in that you need to know what you applying without seeing the result immediately in the builder. I would toggle to see the end result. I am not familiar enough with builder to figure out how to make this work there as well.